src/bosh-director/spec/unit/deployment_plan/network_planner/reservation_reconciler_spec.rb

```diff
@@ -255,7 +255,7 @@ module Bosh::Director::DeploymentPlan
 
             expect(existing_plans[0].reservation.network.name).to eq('my-network-2')
             expect(existing_plans[0].reservation.instance_model).to eq(instance_model)
-            expect(ip_to_netaddr(existing_plans[0].reservation.ip)).to eq('192.168.1.2')
+            expect(ip_to_netaddr(existing_plans[0].reservation.ip).to_s).to eq('192.168.1.2')
```

Comment:
How about using the format_ip method in all these expects with `ip_to_netaddr(..).to_s`

Reply:
Good idea, I changed all of them.